base/common/python/pki/acme.py

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright Red Hat, Inc.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+
+import logging
+import pki.subsystem
+
+logger = logging.getLogger(__name__)
+
+
+class ACMEClient(pki.subsystem.SubsystemClient):
+
+    def __init__(self, parent):
+        super().__init__(parent, 'acme')
+        self.headers = {'Content-type': 'application/json',
+                        'Accept': 'application/json'}
+
+    def enable(self):
+        path = '/acme/enable'
+        response = self.connection.post(path, "", self.headers)
+        logger.debug('Enable ACME returned %d', response.status_code)
```

Comment:
Do you think we should check the response status and return True/False to indicate the outcome, or throw exception on non-200 status?